browser-test/src/application_navigation.test.ts

```diff
@@ -133,12 +134,24 @@ describe('Applicant navigation flow', () => {
     })
 
     it('verify program list page', async () => {
-      const {page} = ctx
+      const {page, adminPrograms} = ctx
+      await loginAsAdmin(page)
+      const programWithExternalLink = 'Program with external link'
+      await adminPrograms.addProgram(
+        programWithExternalLink,
```

Comment:
to be thorough - can you add two case with and without external site.

Reply:
The case without a site already present. (created in beforeAll). Here I'm adding a second program, with external site. So we are testing both here. Or your meant something else?